pkg/cmd/commatrix_test.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/openshift-kni/commatrix/pkg/types"
+	mock_utils "github.com/openshift-kni/commatrix/pkg/utils/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_detectDeploymentAndInfra(t *testing.T) {
```

Comment:
The test focuses on the platform check, should it focus on checking the generate functionality?